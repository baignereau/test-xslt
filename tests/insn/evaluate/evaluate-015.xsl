<xsl:transform xmlns:xsl="http://www.w3.org/1999/XSL/Transform" version="3.0">

  <!-- Basic test of saxon:evaluate-node() -->
  <!-- Formerly saxon013 -->
  <!-- rewritten Nov 2009 to use xsl:evaluate -->

  <xsl:template match="doc">
    <out>
      <xsl:apply-templates/>
    </out>
  </xsl:template>

  <xsl:template match="exp">
    <ans>
      <xsl:evaluate xpath="string(.)" base-uri="{base-uri(.)}" namespace-context="."
        context-item="."/>
    </ans>
  </xsl:template>
  
  <xsl:strip-space elements="*"/>

</xsl:transform>
