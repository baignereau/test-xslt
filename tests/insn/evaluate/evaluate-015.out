<?xml version="1.0" encoding="UTF-8"?>
<out>
  <ans>4</ans>
  <ans>1</ans>
  <ans>eval003.xml</ans>
  <ans>eval003.xml</ans>
  <ans>17</ans>
  <ans>4</ans>
  <ans>1 4 9</ans>
</out>